doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -312,7 +414,7 @@ be changed using the following option.
 .. opt:: Default Proof Mode @string
```

Comment:
The documentation of this command might be worth moving up or at least to its own section (as opposed to a subsection of "Entering and exiting proof mode."

Reply:
You resolved this without changing the location of the command nor the level of the section.

Reply:
Got it.  I don't even recall seeing the comment before.